mantle/kola/harness.go

```diff
@@ -335,6 +340,17 @@ func parseDenyListYaml() error {
 			continue
 		}
 
+		if obj.SnoozeDate != "" {
+			snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
+			if err != nil {
+				return err
+			} else if today.After(snoozeDate) {
+				continue
+			}
+
+			fmt.Printf("🕒 Snoozing test pattern \"%s\" until %s\n", obj.Pattern, snoozeDate.Format("Jan 02 2006"))
+		}
+
 		fmt.Printf("⚠️  Skipping kola test pattern \"%s\":\n", obj.Pattern)
```

Comment:
Feels like we can just print one of those, no? Either we're unconditionally skipping, or we're conditionally snoozing a test.

Reply:
I'm OK with both, but also OK with the way it is. Probably not worth the extra work to reword and conditionalize things.

Reply:
I initially left it because I thought I might over complicate things by introducing extra conditions, but it is true that the output will look more clean. I can add the change.